### PR TITLE
3.9.9

### PR DESCRIPTION
**Pull request by RossKen**
### Type of PR

- [ ] BUG
- [ ] FEAT
- [X] MAINT
- [ ] DOC

### PR Checklist

- [ ] Added documentation for changes
- [ ] Added feature to example notebooks or tutorial (if appropriate)
- [ ] Added tests (if appropriate)
- [X] Updated CHANGELOG.md (if appropriate)
- [ ] Made changes based off the latest version of Splink
- [ ] Run the [linter](https://moj-analytical-services.github.io/splink/dev_guides/changing_splink/lint.html)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [3.9.9] - 2023-11-14
```
Comment (ADBond) [inline CHANGELOG.md:8]: Can we add this under the ## [Unreleased] section so that we still have that for adding new changes?

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,5 +36,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Corrected path for Spark `.jar` file containing UDFs to work correctly for Spark < 3.0 ([#1622](https://github.com/moj-analytical-services/splink/pull/1622))
 - Spark UDF `damerau_levensthein` is now only registered for Spark >= 3.0, as it is not compatible with earlier versions ([#1622](https://github.com/moj-analytical-services/splink/pull/1622))
 
-[unreleased]: https://github.com/moj-analytical-services/splink/compare/v3.9.8...HEAD
-[3.9.8]: https://github.com/moj-analytical-services/splink/compare/v3.9.7...v3.9.8
+[unreleased]: https://github.com/moj-analytical-services/splink/compare/3.9.9...HEAD
+[3.9.9]: https://github.com/moj-analytical-services/splink/compare/v3.9.8...3.9.9
```
Comment (ADBond) [inline CHANGELOG.md:40]: we should keep the old [3.9.8] link as well as adding this new one

**Review by ADBond — APPROVED** [review]
i think there might be some more recent changes that haven't made it into the changelog (whilst we are adopting it - some of the branches were pre-changelog) - would be good to add these (by cross-referencing the github release notes), but that can also be done as a separate PR (and I am happy to do that if so)

**Comment by RossKen** [comment]
> i think there might be some more recent changes that haven't made it into the changelog (whilst we are adopting it - some of the branches were pre-changelog) - would be good to add these (by cross-referencing the github release notes), but that can also be done as a separate PR (and I am happy to do that if so)

Cool, thanks for the comments - I will look out what I can to add to the changelog

**Comment by RossKen** [comment]
@ADBond I wasn't too sure what exactly is of interest in the changelog (user-facing vs internal changes etc) so I have added what I think may be useful for people. Going to merge this now to get the release out, but feel free to add anything else to the changelog in a separate PR if you think they is important.

**Pull request merged**